jetstream/__init__.py

```diff
@@ -5,11 +5,12 @@
 class AnalysisPeriod(enum.Enum):
     DAY = "day"
     WEEK = "week"
+    DAYS_28 = "days_28"
     OVERALL = "overall"
 
     @property
     def adjective(self) -> str:
-        d = {"day": "daily", "week": "weekly", "overall": "overall"}
+        d = {"day": "daily", "week": "weekly", "days_28": "days_28", "overall": "overall"}
```

Comment:
with apologies for the bikeshed: an underscore here makes it a little harder to parse the table names; can we call it `days28`?